26-register_registration/resource_types/research_data/dataset.feature

```diff
@@ -0,0 +1,90 @@
+Feature: Dataset
+
+	In order to document my findings in journal articles
+	As a User
+	I want to be able to publish my own and others datasets
+
+	In order to document my findings in journal articles
+	As a User
+	I want to be able to link published datasets to my published journal articles
+
+	Background:
+		Given User selects Resource type "Research Data"
+		And they select Dataset as subtype
+
+	@9141
+	Scenario: User sees information about data that is illegal to publish on this service
+		When the User has selected to register a Dataset
+		Then the User sees information about data that is illegal to publish on this service
```

Comment:
Foreslår å trekke ut det som går på illegal to publish i en egen Feature. Da kan du legge inn i Background at User prøver å registrere et Dataset som er illegal. Slik det er skrevet nå kan scenarioet tolkes dit at hver gang brukeren registrerer et Dataset så er det illegal.

Reply:
Eg oppfatter problemet, men eg ser ikkje løsningen. Kan du eksemplifisere - det du tenker på?
Det som er viktig i senarioet er at bruker må bekrefte at han (har til intensjon å) publisere eit dataset som inneheld data som han har lov til å publisere. Her er dialogen som bruker skal svare ut:
<img width="1020" alt="image" src="https://user-images.githubusercontent.com/11860616/171866755-fb6343cf-5382-4852-9f4d-94b53bcc923b.png">

Reply:
OK, det gjorde det klarere for meg, hadde misforstått hva som var ment i scenarioet.